bin/create_certs.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+from google.cloud import certificate_manager_v1
+import yaml
+import os
+import sys
+
+def refractr_create_dns_authorization(hostname,id):
+    # Create a client
+    client = certificate_manager_v1.CertificateManagerClient()
+
+    # Initialize request argument(s)
+    dns_authorization = certificate_manager_v1.DnsAuthorization()
+    dns_authorization.domain = hostname
+    dns_authorization.description = "created by script"
+
+    request = certificate_manager_v1.CreateDnsAuthorizationRequest(
+        parent=f"projects/{PROJECT_ID}/locations/global",
+        dns_authorization_id=id,
+        dns_authorization=dns_authorization,
+    )
+
+    # Make the request
+    operation = client.create_dns_authorization(request=request)
+
+    print("Waiting for operation to complete...")
+
+    response = operation.result()
+
+    # Handle the response
+    print(response)
+    return response.name
+
+
+def refractr_create_certificate(hostname,certname, dns_auth=None):
+    # Create a client
+    client = certificate_manager_v1.CertificateManagerClient()
+    certificate = certificate_manager_v1.Certificate()
+    certificate.managed.domains = hostname
+    certificate.managed.dns_authorizations = dns_auth
+    certificate.name = f"projects/{PROJECT_ID}/locations/global/certificates/{certname}"
+
+    # Initialize request argument(s)
+    request = certificate_manager_v1.CreateCertificateRequest(
+        parent= f"projects/{PROJECT_ID}/locations/global",
```

Comment:
```suggestion
        parent= f"projects/{PROJECT_ID}/locations/global",
```

Reply:
shoot I messed up, the change was supposed to be to make it like the other assignments where there is no spacing between the assignment and the varaible like `this=that` instead of `this= that`

Reply:
no worries, so I guess the line 44 needs to be fixed .. which should be completed .

Reply:
yeah that sounds correct that one should be `request = certi...`

Reply:
should be fixed now